src/templates/withDefaultContext.ts

```diff
@@ -1,9 +1,6 @@
+import intl = require('intl');
 import { defaults } from 'lodash';
-
-
-const intl = require('intl');
-const moment = require('moment');
-require('moment-timezone');
+import moment = require('moment');
```

Comment:
```suggestion
import * as moment from 'moment';
```
Not classy :cocktail: